### PR TITLE
US13898: Added Check for supportMultipleGrades

### PR DESCRIPTION
**Pull request by kristian-torres**
## Relevant Rally Links

- https://rally1.rallydev.com/#/289692574792d/search?detail=%2Fuserstory%2F16ee00d8-1857-47a2-8631-8e4f1422befd&keywords=lti

## Goal/Solution

Added a new property on the ContentObjectEntity `supportMultipleGrades` in the attached related PRs. This change adds placeholders for the old/new score editors based on the LTI topic that was selected.

## Screenshots/Video
image

## Related PRs

- [LMS](https://github.com/Brightspace/lms/pull/22755) 
- [Lessons](https://github.com/Brightspace/smart-curriculum/pull/1551)
- [Siren](https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/476)

## Remaining Work
- [x] Dev testing

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-content-editor/lti-link/d2l-activity-content-lti-link-options.js [hunk 1]
```diff
@@ -67,11 +67,28 @@ class ContentEditorLtiLinkOptions extends SkeletonMixin(ErrorHandlingMixin(Local
...
+			return html`
+			<div id="content-link-options-container">
+				<div class="d2l-display-options-text">
+					<label class="d2l-label-text">
+						THIS IS A TEST. IM LTI 1.3 AND I SUPPORT MULTIPLE GRADES
+					</label>
+				</div>
+				${this._renderLinkOptions(isExternalResource)}
+			</div>
+		`;
```
Comment (Kiefer-Co-D2L) [inline components/d2l-activity-editor/d2l-activity-content-editor/lti-link/d2l-activity-content-lti-link-options.js:84]: Nit: I think this line needs to be indented by 1 to match.

File: components/d2l-activity-editor/d2l-activity-content-editor/lti-link/d2l-activity-content-lti-link-options.js [hunk 2]
```diff
@@ -67,11 +67,28 @@ class ContentEditorLtiLinkOptions extends SkeletonMixin(ErrorHandlingMixin(Local
 			? this.entity.isExternalResource
 			: false;
 
+		const supportMultipleGrades = this.entity
```
Comment (ianlaidlaw) [inline components/d2l-activity-editor/d2l-activity-content-editor/lti-link/d2l-activity-content-lti-link-options.js:70]: nit: if you wanted, you could 1-liner this check, but that's just personal preference you may prefer the ternary
```
const suportMultipleGrades = this.entity && this.entity.supportMultipleGrades
``

Comment (kristian-torres) [reply components/d2l-activity-editor/d2l-activity-content-editor/lti-link/d2l-activity-content-lti-link-options.js:70]: I merely just copied this. I can change it to be easier to read. As well as the ternary above.

File: components/d2l-activity-editor/d2l-activity-content-editor/lti-link/d2l-activity-content-lti-link-options.js [hunk 3]
```diff
@@ -63,10 +63,24 @@ class ContentEditorLtiLinkOptions extends SkeletonMixin(ErrorHandlingMixin(Local
...
+						<label class="d2l-label-text d2l-skeletize">
+							${this.localize('content.displayOptions')}
+						</label>
+					</div>
+					${this._renderLinkOptions(isExternalResource)}
+				</div>
+			`;
+		}
+
+		// TODO: Here is where the LTI 1.1 (default) score editor will go. We will return it here.
```
Comment (kristian-torres) [inline components/d2l-activity-editor/d2l-activity-content-editor/lti-link/d2l-activity-content-lti-link-options.js:83]: These are meant for Kiefer who will put the components here and above.

**Review by ianlaidlaw — APPROVED** [review]

**Review by Kiefer-Co-D2L — APPROVED** [review]
LGTM!  We're going to likely have merge conflicts but I can handle them on my end.

**Pull request merged**